tests/network/localnet/liblocalnet.py

```diff
@@ -182,3 +212,71 @@ def client_server_active_connection(
             server_port=port,
         ) as client:
             yield client, server
+
+
+@contextlib.contextmanager
+def create_nncp_localnet_on_secondary_node_nic(
+    nodes_common_available_nic_name: str, mtu: int | None = None
+) -> Generator[libnncp.NodeNetworkConfigurationPolicy, None, None]:
+    """Create NNCP to configure an OVS bridge on a secondary NIC across all worker nodes.
+
+    Note:
+        This function assumes homogeneous hardware—all workers must have a NIC with
+        the same name. The configuration is applied to all workers to support anti-affinity scheduled VMs.
+
+    Args:
+        nodes_common_available_nic_name: The name of common available NIC on all nodes.
+        mtu: Optional MTU to configure on the physical NIC.
+
+    Yields:
+        The created NodeNetworkConfigurationPolicy.
+    """
+    bridge_name = f"localnet-ovs-br-{uuid.uuid4().hex[:16]}"
+    interfaces = []
+
+    if mtu:
+        # Ensure the physical NIC MTU matches the network MTU
+        interfaces.append(
+            libnncp.Interface(
```

Comment:
Can you please confirm that we correctly handle the teardown? I.e. the interface is left working and possibly the mtu value is reverted.